style.css

```diff
@@ -0,0 +1,815 @@
+  padding: 50px 0;
+  letter-spacing: 2px;
+}
+
+.text__title {
+  width: 549px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  line-height: 82px;
+  margin-bottom: 0;
+}
+
+.text__subtitle {
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  letter-spacing: .5px;
+  margin-bottom: 52px;
+}
+
+.collection__next-text {
+  text-decoration: none;
+  color: black;
+  font-style: normal;
+  font-size: 13px;
+  border: 1px solid black;
+  padding: 10px 23px;
+  margin-right: 52px;
+}
+
+.collection__other-text {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 13px;
+  letter-spacing: 0;
+  margin-right: 14px;
+}
+
+.collection__scroll {
+  position: absolute;
+  width: 27px;
+  height: 46px;
+  left: 49%;
+  top: 815px;
+  border: 2px solid #000;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
+
+.collection__scroll-elem {
+  position: absolute;
+  width: 2.45px;
+  height: 8.21px;
+  left: 50%;
+  top: 826.5px;
+  background: #000;
+  border-radius: 19px;
+}
+
+.collection__next-text:hover,
+.collection__other-text:hover {
+  opacity: .5;
+  transition: 1s ease-out .2s;
+}
+
+.vectormob {
+  display: none;
+}
+
+@media (max-width: 1235px) {
+  .first {
+    max-width: 1235px;
+    width: 100%;
+    display: grid;
+    grid-template-columns: 35.2% 64.8%;
+    grid-template-rows: 719px;
+  }
+
+  .header__logo {
+    font-size: 24px;
+    letter-spacing: 2px;
+    margin: 5px 41px;
+  }
+
+  .header__menu {
+    height: 74px;
+    width: 74px;
+  }
+
+  .header__menu-item {
+    height: 3px;
+    width: 28.7px;
+    margin-bottom: 4px;
+  }
+
+  .header__language {
+    left: 29.5%;
+    top: 34px;
+    font-size: 16px;
+  }
+
+  .header__vector {
+    left: 32.5%;
+    top: 42px;
+  }
+
+  .img__img {
+    width: 439px;
+    height: 450px;
+  }
+
+  .step {
+    position: absolute;
+    width: 100%;
+    display: grid;
+    grid-template-columns: 439px 439px;
+    grid-template-rows: 450px;
+    justify-content: center;
+    top: 135px;
+    z-index: 2;
+    grid-column-gap: 42px;
+  }
+
+  .text {
+    height: 450px;
+  }
+
+  .text__title {
+    font-size: 60px;
+    line-height: 68px;
+    letter-spacing: 4px;
+    width: 439px;
+  }
+
+  .text__container {
+    padding: 39px 0;
+  }
+
+  .text__subtitle {
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 1.1px;
+    margin-bottom: 40px;
+  }
+
+  .collection__next-text {
+    font-size: 11px;
+    padding: 10px 27px;
+    margin-right: 43px;
+    letter-spacing: 0px;
+  }
+
+  .collection__other-text {
+    font-size: 11px;
+    letter-spacing: 0px;
+    margin-right: 15px;
+  }
+
+  .collection__vector {
+    width: 35px;
+  }
+
+  .collection__scroll {
+    width: 22px;
+    height: 39px;
+    left: 46.5%;
+    top: 650px;
+  }
+
+  .collection__scroll-elem {
+    left: 47.2%;
+    top: 659.5px;
+  }
+
+  .first__sidebar-social-inst, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)